src/main/groovy/com/twcable/grabbit/jcr/ProtoPropertyDecorator.groovy

```diff
@@ -82,20 +82,6 @@ class ProtoPropertyDecorator {
         innerProtoProperty.name == JCR_MIXINTYPES
     }
 
-
-/*
-    boolean isMultiple() {
```

Comment:
It will now use the `isMultiple()` method from the generated Java class from the updated node.proto message